src/pages/Admin/Admin.tsx

```diff
@@ -286,6 +286,10 @@ const VenueInfoComponent: React.FC<VenueInfoComponentProps> = ({
   const match = useRouteMatch();
   const placementDivRef = useRef<HTMLDivElement>(null);
 
+  const navigateToAdmin = useCallback(() => {
+    history.push("/admin");
```

Comment:
Like there is `adminNGRootUrl`, one can be made for OG admin:

```suggestion
    history.push(adminOGRootUrl);
```

Reply:
New PR is a good idea, on top of these constants, there might be a need to switch between OG/NG once v3 achieves v1 parity